packages/url/src/filter-url-for-display.js

```diff
@@ -13,8 +13,14 @@
  * @return {string} Displayed URL.
  */
 export function filterURLForDisplay( url, maxLength = null ) {
+	if ( ! url ) {
+		return '';
+	}
```

Comment:
Worth adding a test case for this one, too.

Reply:
Added the test case for it 🙇